cmd/algod/main.go

```diff
@@ -281,12 +281,12 @@ func run() int {
 
 		// make sure that the format of each entry is valid:
 		for idx, peer := range peerOverrideArray {
-			url, err := network.ParseHostOrURL(peer)
-			if err != nil {
+			addr, addrErr := network.ParseHostOrURLOrMultiaddr(peer)
+			if addrErr != nil {
 				fmt.Fprintf(os.Stderr, "Provided command line parameter '%s' is not a valid host:port pair\n", peer)
 				return 1
 			}
-			peerOverrideArray[idx] = url.Host
+			peerOverrideArray[idx] = addr
```

Comment:
peerOverrideArray could include both multiaddr and  url.Host. will nodes be able to resolve both of these addresses?

Reply:
I think logging an error is preferable to bringing the node down for a single typo in a single manual entry. 

I have a PR against this branch that potentially addresses this: https://github.com/cce/go-algorand/pull/15 but I guess it's not useful at all if we implement a third GossipNode for the hybrid version